encoding/openapi/crd.go

```diff
@@ -63,6 +63,34 @@ func (b *builder) coreSchemaWithName(name string) *ast.StructLit {
 	return s
 }
 
+func (b *builder) isUnstructured() bool {
+	path := b.ctx.path
+	if len(path) == 0 {
+		return false
+	}
+	if path[len(path)-1] == "*" {
```

Comment:
why are we stripping trailing asterisks?

Reply:
this is for the case when the field is repeated. i'll add a comment